jenkins/node-image/node-runner.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Copyright 2015 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A wrapper script for running node-e2e tests
+
+export PATH=${WORKSPACE}/google-cloud-sdk/bin:${PATH}
+
+# Update gcloud
+gcloud components update
+
+# Authenticate gcloud
+if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
+  gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
+fi
+
+
+# Run script
+NODE_SCRIPT=$1
```

Comment:
rather than passing these as arguments, you could just inject these env vars into the docker environment directly, by adding appropriate `-e` arguments to `docker run`. this probably lines up with the eventual job env var stuff nicely.